lib/service/Service.ts

```diff
@@ -121,17 +119,14 @@ class Service extends EventEmitter {
     const { currency } = args;
     const balances = new Map<string, { balance: number, pendingOpenBalance: number }>();
     const getBalance = async (currency: string) => {
-      const client = this.lndClients[currency.toUpperCase()]
-        // TODO: support all registered tokens for Raiden
-        || (currency.toUpperCase() === 'WETH' && this.raidenClient);
-
-      if (!client) {
-        // TODO: throw an error here indicating that lnd is disabled for this currency
+      try {
+        const client = this.swapClientsManager.get(currency.toUpperCase());
```

Comment:
I think best to call this `swapClient` just to be consistent with how its called elsewhere